docs/sections/userQuickStart.md

```diff
@@ -0,0 +1,131 @@
+<link rel="stylesheet" href="{{baseUrl}}/css/main.css">
+
+<include src="../common/header.md" />
+
+<div class="website-content">
+
+# User Quick Start
+
+## Requirement
+We expect users to have basic knowledge of the following:
+- [x] Markdown
+- [x] HTML & CSS
+- [x] npm (Node.js)
+- [x] Command-line Environment
+
+## Installation
+`MarkBind` can be installed using npm
+
+`v1.4.3`: and above: Node.js with version 8.0 and higher is required.
+`v1.4.2`: We recommend users to use Node.js with version 6.5 or higher.

```

Comment:
No need to document older versions. Future users will not be using those old versions anyway.